impeller/display_list/vertices_converter.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include "flutter/display_list/display_list.h"
+#include "flutter/display_list/display_list_vertices.h"
+#include "impeller/geometry/vertices.h"
+
+namespace impeller {
+
+class VerticesConverter {
+ public:
+  VerticesConverter() = default;
+
+  ~VerticesConverter() = default;
+
+  Vertices ToVertices(const flutter::DlVertices* vertices);
```

Comment:
I like the idea of having this in a separate TU since the implementation is getting complicated, but since this has no state and everything is public, maybe just get rid of the class with the constructor/destructor and just declare this as a function in the impeller namespace?

Reply:
Will do!